ethicml/algorithms/inprocess/mlp.py

```diff
@@ -14,10 +14,10 @@
 
 
 ACTIVATIONS: Dict[str, ActivationType] = {
-    "identity": "identity",
-    "logistic": "logistic",
-    "tanh": "tanh",
-    "relu": "relu",
+    "identity": ActivationType.identity,
+    "logistic": ActivationType.logistic,
+    "tanh": ActivationType.tanh,
+    "relu": ActivationType.relu,
 }
```

Comment:
This can just be replace with `ActivationType[some_string]`, I think.

Reply:
I just removed this, I don't think it's useful